src/App.js

```diff
@@ -1,24 +1,15 @@
-import logo from './logo.svg';
 import './App.css';
+import Header from './components/header/Header.js';
+import Description from './components/description/Description.js';
+import MainContent from './components/mainContent';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+<>
```

Comment:
Could you fix indentation across all files? Use prettier for it. It should be installed with CRA?